### DISCUSSION TITLE
Configurable Kitchen Screen

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#398] [Ideas]
Hi Team,

I have configured the printer for a kitchen. And I tried to send an order to the kitchen. I do notice that the order is printed on a printer as well it is available on the kitchen screen. We can improve this

1) Give an option to enable the kitchen screen for the kitchen instead of making it enabled by default.

or

1) Disable kitchen screen if the printer is enabled

Some people may need both and some may need only one.

**Comment by Blair2004** [comment]
Hi, we'll rather add an option to disable the kitchen screen.
Thank you for this valuable contribution.
